src/activity_actor.cpp

```diff
@@ -3897,6 +3900,82 @@ std::unique_ptr<activity_actor> haircut_activity_actor::deserialize( JsonIn & )
     return haircut_activity_actor().clone();
 }
 
+void pry_nails_activity_actor::start( player_activity &act, Character & )
+{
+    act.moves_total = moves_total;
+    act.moves_left = moves_total;
+}
+void pry_nails_activity_actor::do_turn( player_activity &act, Character &who )
+{
+    sfx::play_activity_sound( "tool", "hammer", sfx::get_heard_volume( act.placement ) );
+}
+void pry_nails_activity_actor::finish( player_activity &act, Character &who )
+{
+    const ter_id type = here.ter( target );
+
+    int nails = 0;
+    int boards = 0;
+    ter_id newter;
+    if( type == t_fence ) {
+        nails = 6;
+        boards = 3;
+        newter = t_fence_post;
+        who.add_msg_if_player( _( "You pry out the fence post." ) );
+    } else if( type == t_window_boarded ) {
+        nails = 8;
+        boards = 4;
+        newter = t_window_frame;
+        who.add_msg_if_player( _( "You pry the boards from the window." ) );
+    } else if( type == t_window_boarded_noglass ) {
+        nails = 8;
+        boards = 4;
+        newter = t_window_empty;
+        who.add_msg_if_player( _( "You pry the boards from the window frame." ) );
+    } else if( type == t_door_boarded || type == t_door_boarded_damaged ||
+               type == t_rdoor_boarded || type == t_rdoor_boarded_damaged ||
+               type == t_door_boarded_peep || type == t_door_boarded_damaged_peep ) {
+        nails = 8;
+        boards = 4;
+        if( type == t_door_boarded ) {
+            newter = t_door_c;
+        } else if( type == t_door_boarded_damaged ) {
+            newter = t_door_b;
+        } else if( type == t_door_boarded_peep ) {
+            newter = t_door_c_peep;
+        } else if( type == t_door_boarded_damaged_peep ) {
+            newter = t_door_b_peep;
+        } else if( type == t_rdoor_boarded ) {
+            newter = t_rdoor_c;
+        } else { // if (type == t_rdoor_boarded_damaged)
+            newter = t_rdoor_b;
+        }
+        who.add_msg_if_player( _( "You pry the boards from the door." ) );
+    }
+    who.practice( skill_fabrication, 1, 1 );
```

Comment:
Prying out nails makes you better at fabrication? I understand you're just copied the old code, but I think this is just nonsense and should be removed.

Reply:
yeah it's a typo. Thanks!

Reply:
You could make a case that seeing how someone laid out nails before would make you better at laying nails in the future! But we can just remove it too, I don't think it would change much.